sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/QueryStageExec.scala

```diff
@@ -140,6 +140,8 @@ case class ShuffleQueryStageExec(
       case _ =>
     }
   }
+
+  def supportAdaptive: Boolean = plan.supportAdaptive
```

Comment:
:nit -> `exchangeSupportAdaptive`? 
`QueryStage` is already a part of adaptive execution